examples/logging/tutorial-custom-logger/README.md

```diff
@@ -0,0 +1,31 @@
+# Custom Logger
+
+This page explains how to create a Custom Logger for use with DeepSparse Logging.
+
+### Why Might I Need A Custom Logger?
```

Comment:
Do we need to motivate the need for a custom logger through the lack of features on the Prometheus side? I'd say the sufficient motivation is user empowerment to log anything anywhere